hijack/static/hijack/hijack-styles.css

```diff
@@ -1,85 +0,0 @@
-.hijacked-warning {
```

Comment:
Whithout that file it's not possible to just `pip install git+https://github.com/django-hijack/django-hijack.git@build` and hope that everything will work.

Reply:
OK, that's true. I dished the `Makefile` for a proper setuptools integration. How you can install it via git, however, you will get an error if either gettext or node is missing… which IMHO is OK. I also extended the contributing guide to point to that fact.